File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.cpp
class Solution {
public:
    int maxProduct(vector<int>& nums) {
        sort(nums.begin(),nums.end(),greater<int>());
        int first = 0;
        int last = 1;
        int result = (nums[first]-1)*(nums[last]-1);
        return result;
    }
};

File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.java
class Solution {
    public int maxProduct(int[] nums) {
        int res = 0;
        Arrays.sort(nums);
        for(int i = nums.length-1;i>0;i--){
            int j = i - 1;
            int temp = (nums[i]-1) * (nums[j]-1);
            res = Math.max(res,temp);
        }
        return res;
    }
}